Applications/Spreadsheet/main.cpp

```diff
@@ -175,41 +191,30 @@ int main(int argc, char* argv[])
     },
         window));
     edit_menu.add_action(GUI::CommonActions::make_paste_action([&](auto&) {
+        ScopeGuard update_after_paste { [&] { spreadsheet_widget.current_worksheet().update(); } };
+
         auto& cells = spreadsheet_widget.current_worksheet().selected_cells();
         ASSERT(!cells.is_empty());
         const auto& data = GUI::Clipboard::the().data_and_type();
         if (auto spreadsheet_data = data.metadata.get("text/x-spreadsheet-data"); spreadsheet_data.has_value()) {
```

Comment:
Looking at how this is used elsewhere, I don't think `Clipboard::DataAndType` is meant to contain different types of data - just one and its metadata.
while this works just fine for my purposes, I'm almost tempted to switch that up to use `MimeData` too, should be alright since there's a raw bitmap encoding now, I think?

Reply:
Yes, the `Clipboard::DataAndType` is vestigial now and should be replaced with `MimeData` :)